src/NUnitFramework/tests/Internal/ExceptionHelperOutputExceptionDataTests.cs

```diff
@@ -0,0 +1,152 @@
+// ***********************************************************************
+// Copyright (c) 2019 Charlie Poole
```

Comment:
One copyright header and Charlie Poole, Rob Prouse 😄

Reply:
Fixed.  I have no idea how the duplicates happened since that was copy/paste.

Reply:
@rprouse @drewburlingame I bet this is the problem:

https://github.com/nunit/nunit/blob/edb0abc185af3e780e835bbb76318154e0b13091/NUnit.sln.DotSettings#L85-L105